articles/flows/guides/regular-web-app-login-flow/includes/sample-use-cases-add-login.md

```diff
@@ -12,7 +12,7 @@ https://${account.namespace}/authorize?
     scope=openid
 ```
 
-Now, when you [request tokens](/api-auth/tutorials/regular-web-app-login-flow/request-tokens), your ID Token will contain the most basic claims. When you [decode the ID Token](/tokens/id-token#id-token-payload), it will look similar to:
+Now, when you [request tokens](#request-tokens), your ID Token will contain the most basic claims. When you [decode the ID Token](/tokens/id-token#id-token-payload), it will look similar to:
 
```

Comment:
Link to request tokens doesn't seem to work for me

Reply:
Thanks. These are includes, so they were working inside the full tutorial, but not from the include files. Fixed it!